security/audit-log-view.adoc

```diff
@@ -36,5 +36,5 @@ ifndef::openshift-rosa,openshift-dedicated[]
 * xref:../support/gathering-cluster-data.adoc#about-must-gather_gathering-cluster-data[Must-gather tool]
 * link:https://github.com/kubernetes/apiserver/blob/master/pkg/apis/audit/v1/types.go#L72[API audit log event structure]
 * xref:../security/audit-log-policy-config.adoc#audit-log-policy-config[Configuring the audit log policy]
-* xref:../logging/log_collection_forwarding/log-forwarding.adoc#log-forwarding[About log forwarding]
+* xref:../observability/logging/log_collection_forwarding/log-forwarding.adoc#log-forwarding[About log forwarding]
```

Comment:
Actual section titled "About log collection and forwarding"

Reply:
Please see my https://github.com/openshift/openshift-docs/pull/74819#discussion_r1577387726.